app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -1103,6 +1106,16 @@ def build_toolbar_disable_button(id)
       case id
       when "policy_delete"
         return "Policies that belong to Profiles can not be deleted" if @policy.memberof.length > 0
+        return "This policy is read only and cannot be deleted" if @policy.read_only
```

Comment:
There should be no additions to this file. All the new button behavior should go under `app/helpers/application_helper/button/`.
The code that is already here will be also converted to separate buttons in the future.